packages/lesswrong/server/migrations/2022-08-19-createPodcastsForPosts.ts

```diff
@@ -0,0 +1,62 @@
+import { registerMigration } from './migrationUtils';
+import { Podcasts } from '../../lib/collections/podcasts/collection';
+import { PodcastEpisodes } from '../../lib/collections/podcastEpisodes/collection';
+import { Posts } from '../../lib/collections/posts/collection';
+
+import razPostToBuzzsproutMappings from './resources/razPostToBuzzsproutMappings.json';
+import curatedPostToBuzzsproutMappings from './resources/curatedPostToBuzzsproutMappings.json';
```

Comment:
Back in the day this would have been in js. Thank you 🙏

Reply:
Speak of the devil: https://github.com/ForumMagnum/ForumMagnum/pull/5512

Reply:
Convince me that being in the js is terrible?

Reply:
Actually I probably usually favor defining things in-line, purely for the type info, though importing JSON now (mostly) gives you the same type info (though this is a recent improvement in TS, I think).  In this case they were just way too big.